angular/src/app/app.component.html

```diff
@@ -1,5 +1,27 @@
-<h1>Hello {{title}}! <calcite-icon icon="banana"></calcite-icon></h1>
+<style>
+  calcite-icon {
+    --calcite-ui-icon-color: #ffe135;
+  }
+</style>
 
-<calcite-combobox placeholder="Select a field" label="Select a field" (calciteComboboxChange)="logSelectedFields($event)">
-  <calcite-combobox-item *ngFor="let field of fields" [value]="field.value" [textLabel]="field.displayText"></calcite-combobox-item>
-</calcite-combobox>
+<div *ngIf="isLoading; else content">
```

Comment:
Minor: I don't know if you need the ";" or if that's a typo? I hadn't seen using a semi-colon before.

Reply:
It may be a typo, I initially copy/pasted from the ArcGIS Web Components sample, which is [where that came from](https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/blob/e0b5ba08e2889720f3540d45274897f267d4f5ef/packages/coding-components/templates/arcade-editor/angular/src/app/app.component.html#L14). I'll remove it, thanks!

cc @omarkawach

Reply:
@odoe wrote the Angular sample. Rene WDYT?

Reply:
Ignore my deleted comment. I did in fact write that. I'll take a look.

Reply:
Nevermind, I see it in the Angular doc here: https://angular.io/api/common/NgIf#showing-an-alternative-template-using-else. #justlearnedsomething